### DISCUSSION TITLE
Práctica VUE

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#504] [Q&A]
Buenas tardes, ¿Alguien conoce algún curso corto, lectura o un video para entender mejor la práctica?

**Comment by FabioMurillo16** [comment]
Espero te sirva https://es.vuejs.org/v2/cookbook/client-side-storage.html

**Reply by GloriaCoralCerecedo** [reply]
Gracias!
